### ISSUE TITLE
Use external theory predictions

### ISSUE THREAD

**Issue by tilmantroester** [LSSTDESC/TJPCov#40]
As far as I can tell (please correct me if I'm wrong), TJPCov current computes the theory Cls using CCL, given a `ccl.Cosmology` object. Is there support to supply these Cls directly, e.g. through a sacc file? Computing these quantities here duplicates the functionality in TXPipe and, more importantly, firecrown.

This duplication is not good for maintainability. For example, this hack isn't necessary anymore and should never have been used in the first place:
https://github.com/LSSTDESC/TJPCov/blob/master/tjpcov/main.py#L462

**Comment by carlosggarcia** [comment]
I agree, it should allow it. Although there must be some level of duplicity if we also want TJPCov to be usable as a stand-alone application.
